test/e2e/fixtures.go

```diff
@@ -304,8 +319,10 @@ func exportLogs(tb testing.TB, data *TestData, logsSubDir string, writeNodeLogs
 }
 
 func teardownFlowAggregator(tb testing.TB, data *TestData) {
-	if err := testData.gracefulExitFlowAggregator(testOptions.coverageDir); err != nil {
-		tb.Fatalf("Error when gracefully exiting Flow Aggregator: %v", err)
+	if testOptions.enableCoverage {
```

Comment:
Thanks for adding this missing code as suggested, so we do not fail for tests that don't use flow aggregator coverage binary like ipv6 tests.